docs/status.md

```diff
@@ -113,7 +113,7 @@ one of the following tracking labels.
 | outfeed                  | no            | revisit      | no             | no              | no          |
 | pad                      | yes           | yes          | yes            | yes             | no          |
 | popcnt                   | yes           | yes          | yes            | yes             | no          |
-| power                    | no            | yes*         | yes*           | yes             | no          |
+| power                    | yes           | yes          | yes            | yes             | no          |
```

Comment:
power is also one of those StableHLO_BinaryElementwiseOp where it does not have complete verification check for supported types. This should be mark `revisit` with a link to an issue addressing this.

Reply:
I checked it fails for i1 during compilation

Reply:
Other types are allowed AFAIR

Reply:
On the StableHLO side?

Reply:
stablehlo-opt allows i1, need to add a ticket, thanks

Reply:
#513